modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminManagementToolbarDisplayContext.java

```diff
@@ -397,10 +396,6 @@ public int getTotal() {
 	}
 
 	public List<ViewTypeItem> getViewTypeItems() {
-		if (!FeatureFlagManagerUtil.isEnabled("LPS-169675")) {
```

Comment:
from `portal-impl` you should run : `ant format-source-current-branch`

To remove the entry on the portal.properties